apps/openbadges-system/CLAUDE.md

```
@@ -0,0 +1,252 @@
+# Stop containers
+bun run docker:down
+
+# View logs
+bun run docker:logs
+```
+
+## Architecture Overview
+
+### Directory Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)